pkg/backup/storage.go

```diff
@@ -124,17 +132,17 @@ func (s *S3BackupStorage) List(ctx context.Context) ([]string, error) {
 
 func (s *S3BackupStorage) Push(ctx context.Context, fileName string) error {
 	filePath := filepath.Join(s.basePath, fileName)
-	_, err := s.client.FPutObject(ctx, s.bucket, fileName, filePath, minio.PutObjectOptions{})
+	_, err := s.client.FPutObject(ctx, s.bucket, s.Prefix+fileName, filePath, minio.PutObjectOptions{})
```

Comment:
Needs to be added to the `List` operation as well.

Reply:
Oh, missed that completely. Added it to the `ListObjectsOptions`.